content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -320,6 +320,19 @@ To update Pods to a new spec in a controlled way, use a
 [Deployment](/docs/concepts/workloads/controllers/deployment/#creating-a-deployment), as
 ReplicaSets do not support a rolling update directly.
 
+### Terminating Pods
+
+{{< feature-state feature_gate_name="DeploymentPodReplacementPolicy" >}}
+
+This feature is disabled by default. You can enable it by setting the `DeploymentPodReplacementPolicy`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/)
+and on the [kube-controller-manager](/docs/reference/command-line-tools-reference/kube-controller-manager/)
+
+Pods that become terminating due to deletion or scale down may take a long time to terminate, and may consume
+additional resources during that period. As a result, the total number of all pods can temporarily exceed
+`.spec.replicas`. Terminating pods can be tracked using the `.status.terminatingReplicas` field.
```

Comment:
Is the 'stats.terminatingReplicas' on a ReplicaSet or a Deployment?

Reply:
Thanks! I  made that clearer.

I also saw that we mention the `enabled by default` in the `< feature-state >`, so I have removed that part from the text